.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
did you mean to include this?

Reply:
no I wanted to exclude it to make sure I don't mess the repo up with some GoLand specific stuff